desktop/assets/texts/tr-TR/main_menu.ftl

```diff
@@ -34,7 +34,7 @@ debug-menu = Hata Ayıklama Araçları
 debug-menu-open-stage = Sahne Bilgilerini Göster
 debug-menu-open-movie = Filmi Göster
 debug-menu-open-movie-list = Bilinen Filmleri Göster
-debug-menu-open-domain-list = Domainleri Görüntüle
+debug-menu-open-domain-list = Alan Adlarını Görüntüle
```

Comment:
The new version seems to mean more like "domain names" (as in web addresses), which is not what we mean here.